src/oca_github_bot/version_branch.py

```diff
@@ -9,7 +9,7 @@
     r"-ocabot-merge"
     r"-pr-(?P<pr>\d+)"
     r"-by-(?P<username>\S+)"
-    r"-bump-(?P<bumpversion_mode>(no|patch|minor|major))"
+    r"-bump-(?P<bumpversion_mode>(nobump|patch|minor|major))"
```

Comment:
Can we keep this unchanged? This is not necessary and would avoid issues if we upgrade the bot while merges are in progress.

Likewise I'd prefer keeping the rest of the code unchanged and modify only the command option parsing. In the code, it's good to have `bumpversion_mode=None` when we don't want to bump version, and this tranformation from "nobump" to None is easily done in the options parser.

Reply:
Hum. You're right. It was simplifying all the code, removing some if, but there is a risk to break some command during the process. I will roll back some changes so.
thanks for this point.